tox.ini

```diff
@@ -68,12 +68,12 @@ commands =
 [testenv:py27-mysql]
 basepython = python2.7
 setenv =
-    SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://mysqluser:mysqluserpassword@localhost/superset?charset=utf8
+    SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://root@localhost/superset?charset=utf8
```

Comment:
Why are we changing the user/password for the test databases?

Reply:
It didn't work with the previous credentials. I think insufficient privileges for adding a table if I recall correctly.